From 4131877b0b705202debf2c98ba0fa9fd618ffaf1 Mon Sep 17 00:00:00 2001
From: Rob Herring <r.herring@freescale.com>
Date: Fri, 8 May 2009 15:28:15 -0500
Subject: [PATCH 058/280] ENGR00110997: Change mx51 platform name to imx51

Change mx51 platform name to imx51 so libs will build for all
mx51 platforms.
Remove old MXC platform stuff.

Signed-off-by: Rob Herring <r.herring@freescale.com>
---
 sahara2/Makefile | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/sahara2/Makefile b/sahara2/Makefile
index a5a2dfc..9c2be69 100644
--- a/sahara2/Makefile
+++ b/sahara2/Makefile
@@ -7,22 +7,10 @@ INC := $(INCLUDE)
 
 DIAGS=
 
-ifeq ("$(PLATFORM)", "MXC27530EVB")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91231
-endif
-ifeq ("$(PLATFORM)", "MXC30030ADS")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91321
-endif
-ifeq ("$(PLATFORM)", "MXC30030EVB")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91321
-endif
 ifeq ("$(PLATFORM)", "IMX27ADS")
 TARGET_ARCH=-DCONFIG_ARCH_MX27
 endif
-ifeq ("$(PLATFORM)", "MXC30031ADS")
-TARGET_ARCH=-DCONFIG_ARCH_MXC92323
-endif
-ifeq ("$(PLATFORM)", "IMX51_3STACK")
+ifeq ("$(PLATFORM)", "IMX51")
 TARGET_ARCH=-DCONFIG_ARCH_MX51
 endif
 
-- 
1.8.0

